install/mysql/glpi-empty.sql

```diff
@@ -10062,16 +10064,16 @@ DROP TABLE IF EXISTS `glpi_assets_customfielddefinitions`;
 CREATE TABLE `glpi_assets_customfielddefinitions` (
   `id` int unsigned NOT NULL AUTO_INCREMENT,
   `assets_assetdefinitions_id` int unsigned NOT NULL,
-  `name` varchar(255) NOT NULL,
+  `system_name` varchar(255) NOT NULL,
   `label` varchar(255) NOT NULL,
   `type` varchar(255) NOT NULL,
   `field_options` json,
   `itemtype` VARCHAR(255) NULL DEFAULT NULL,
   `default_value` text,
   `translations` JSON NOT NULL,
   PRIMARY KEY (`id`),
-  UNIQUE KEY `unicity` (`assets_assetdefinitions_id`, `name`),
-  KEY `name` (`name`)
+  UNIQUE KEY `unicity` (`assets_assetdefinitions_id`, `system_name`),
+  KEY `system_name` (`system_name`)
```

Comment:
It should be a unique key. People may try to add definitions using SQL queries.

```suggestion
  UNIQUE KEY `system_name` (`system_name`)
```

Reply:
This is already handled by the key above it, no? Custom field system names are only required to be unique for the asset definition they are for.